libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -98,17 +107,84 @@ private GutenbergContainerFragment getGutenbergContainerFragment() {
         return mRetainedGutenbergContainerFragment;
     }
 
+    /**
+     * Returns the gutenberg-mobile specific translations
+     *
+     * @return Bundle a map of "english string" => [ "current locale string" ]
+     */
+    public Bundle getTranslations() {
+        Bundle translations = new Bundle();
+        Locale defaultLocale = new Locale("en");
+        Resources currentResources = getResources();
+        Configuration configuration = currentResources.getConfiguration();
```

Comment:
Nit, but not it makes debugging this method easier: Let's rename this variable to `currentConfiguration` instead of just `configuration` and then at line 128 let's create a new one instead of reassigning the old one. Thanks!